docs/user_workloads/accessing_virtual_machines.md

```diff
@@ -242,52 +242,55 @@ spec:
 EOF
 ```
 
-### Accessing the VMI using virtctl
+### Accessing a VM or VMI using virtctl
 
-The user can create a websocket backed network tunnel to a port inside the
-instance by using the `virtualmachineinstances/portforward` subresource of the
-`VirtualMachineInstance`.
+The user can create a websocket backed network tunnel to a port inside an
+instance by using the `virtualmachineinstances/portforward` subresource of a
+`VirtualMachineInstance` or the `virtualmachines/portfoward` subresource of a
+`VirtualMachine`.
 
-One use-case for this subresource is to forward SSH traffic into
-the `VirtualMachineInstance` either from the CLI or a web-UI.
+One use-case for these subresources is to forward SSH traffic into
+a `VirtualMachineInstance` either from the CLI or a web-UI. The `VirtualMachine`
+subresource will automatically look up the corresponding `VirtualMachineInstance`.
 
 To connect to a `VirtualMachineInstance` from your local machine, `virtctl`
-provides a lightweight SSH client with the `ssh` command, that uses port
+provides an SSH client with the `ssh` command, that uses port
 forwarding. Refer to the command's help for more details.
 
 ```shell
 virtctl ssh
 ```
 
 To transfer files from or to a `VirtualMachineInstance` `virtctl` also
-provides a lightweight SCP client with the `scp` command. Its usage is
-similar to the `ssh` command. Refer to the command's help for more details.
+provides a SCP client with the `scp` command. Its usage is similar to the `ssh` command.
+Refer to the command's help for more details.
 
 ```shell
 virtctl scp
 ```
 
 #### Using virtctl as proxy
 
-If you prefer to use your local OpenSSH client, there are two ways of doing
-that in combination with virtctl.
+The preferred way to connect to VMs and VMIs is to let wrap local OpenSSH clients.
```

Comment:
"let wrap local OpenSSH clients."
Seems like there's a word missing.
Are you saying the preferred way to connect to VMs and VMIs is to use `virtctl` to wrap local OpenSSH clients?

Reply:
Yes, that's what I wanted to say. Updated it.